chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -263,8 +263,54 @@ BravePrepopulatedEngineID GetDefaultSearchEngine(int country_id, int version) {
           {country_codes::CountryCharsToCountryID('U', 'Z'),
            PREPOPULATED_ENGINE_ID_YANDEX},
       });
-
-  if (version > 16) {
+  static const base::NoDestructor<
+      base::flat_map<int, BravePrepopulatedEngineID>>
```

Comment:
could be `constexpr fixed_flat_map`?

Reply:
i mean, using `base::MakeFixedFlatMap`

Reply:
done, thanks!

Reply:
Nice! Didn't know about `base::MakeFixedFlatMap` 😄👍